wire-ios/Wire-iOS/Sources/Helpers/String+Capitalization.swift

```diff
@@ -0,0 +1,38 @@
+//
+// Wire
+// Copyright (C) 2023 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+
+extension String {
+    /// The first character in each sentence changed to its corresponding uppercase value.
+    /// All remaining characters set to their corresponding lowercase values.
+    ///
+    /// Caveat:
+
+    ///  1. The implementation only works with a string that contains one sentence.
+    ///  The second sentence won't get capitalized.
+    ///
+    ///  2. It doesn't work if the first string is a whitespace character and delimiter.
+    var capitalizedCharacter: String {
```

Comment:
We have it already:
```
extension String {
    func capitalizingFirstLetter() -> String {
        let first = String(prefix(1)).capitalized
        let other = String(dropFirst())
        return first + other
    }
}
```

Reply:
I will keep the more recent implementation (aka the one i introduced and I will get rid of that one you posted). It's not being used in many places and also it doesn't make sure that we lowercase the rest of the letters

Reply:
You could also make use of this:

```swift
extension String {

  var capitalizingFirstLetterOnly: String {
    return lowercased.capitalizingFirstLetter()
  }

}
```

Reply:
This is not relevant anymore that's why I'm marking it as resolved.